checkbox-provider-ce-oem/units/crypto/accelerator.pxu

```diff
@@ -48,6 +62,8 @@ command:
         echo "FAIL: CAAM job ring interrupts didn't increase!"
         exit 1
     fi
+requires: manifest.has_caam == 'True'
+imports: from com.canonical.plainbox import manifest
```

Comment:
Could you help to put this 2 lines to the upper of this job? To make the `command:` field be the bottom of the test job. :bow:

Reply:
done. Thanks